cmake/SetupBlaze.cmake

```diff
@@ -57,6 +57,44 @@ if(SLEEF_FOUND)
   set(_BLAZE_USE_SLEEF 1)
 endif()
 
+# If BLAZE_USE_STRONG_INLINE=ON, Blaze will use this keyword to increase the
+# likelihood of inlining. If BLAZE_USE_STRONG_INLINE=OFF, uses inline keyword
+# as a fallback.
+option(BLAZE_USE_STRONG_INLINE "Increase likelihood of Blaze inlining." ON)
+
+if("${BLAZE_USE_STRONG_INLINE}" STREQUAL "ON")
```

Comment:
I think all this can just be:

```
set(_BLAZE_USE_STRONG_INLINE 0)

if(BLAZE_USE_STRONG_INLINE)
  set(_BLAZE_USE_STRONG_INLINE 1)
endif()
```

Same below

Reply:
Wow I was concerned about the removal of the error handling but am impressed how it is able to handle other flag values like `yes`, `NO`, `false`, etc. so intuitively

Reply:
Fixed in new fixup